charts/opentelemetry-kube-stack/values.yaml

```diff
@@ -355,12 +426,113 @@ defaultCRConfig:
 # This configuration allows for multiple layers of overrides for different clusters. For example, you could
 # create a collector called test with an OTLP exporter in your values.yaml, and then override the endpoint's
 # destination in a file called values-staging.yaml.
-collectors: {}
+collectors:
+  daemon:
+    name: daemon
+    mode: daemonset
+    enabled: true
+    resources:
+      limits:
+        cpu: 100m
+        memory: 250Mi
+      requests:
+        cpu: 100m
+        memory: 128Mi
+    # A scrape config file to instruct the daemon collector to pull metrics from any matching targets on the same node with
+    # prometheus.io/scrape=true
+    # This config also scrapes a running node exporter and the kubelet CAdvisor metrics which aren't currently supported.
+    scrape_configs_file: "daemon_scrape_configs.yaml"
+    presets:
+      logsCollection:
+        enabled: true
+      kubernetesEvents:
```

Comment:
a daemonset shouldn't be used to collect events, I believe it will result in duplicate data. Should be used in the cluster instance.